src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/TextEditingTargetWidget.java

```diff
@@ -575,6 +576,23 @@ public void run()
       toolbar.addRightSeparator();
       toolbar.addRightWidget(toggleDocOutlineButton_);
       
+      showWhitespaceCharactersCheckbox_ = new CheckBox("Show whitespace characters");
```

Comment:
How about just `Show whitespace`?

Also, we may need some logic to hide this control entirely when the editor is very narrow so that it doesn't overflow, wrap, or push other controls off the palette (if you've already experimented with this then disregard!)

Reply:
That's a good idea. My thought here was that shortening the checkbox name isn't worth it since we don't insert many tool icons for `.csv` / `.tsv` icons, though.